### PR TITLE
{bio}[foss/2018b] GenomeTools v1.5.10

### PR DESCRIPTION
**Pull request by nathanhaigh**

### CODE REVIEW

File: easybuild/easyconfigs/g/GenomeTools/GenomeTools-1.5.10-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/genometools/genometools/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['a6aa7f158a3cef90fea8d0fe24bfad0c3ee96b17b3ba0c1f6462582593af679e']
+
+dependencies = [
+    ('cairo', '1.14.12'),
+    ('Pango', '1.40.14'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/GenomeTools/GenomeTools-1.5.10-foss-2018b.eb]: Is there a reason you're not using the existing Pango 1.42.4 we already have for foss/2018b?

Comment (boegel) [reply easybuild/easyconfigs/g/GenomeTools/GenomeTools-1.5.10-foss-2018b.eb]: @akesandgren Most likely that's just an oversight, we can take over the reigns on this one by bumping this to 1.42.4 and getting rid of the `Pango` easyconfig being added here?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/3a3cea3df3b116831f597a89041c905d for a full test report.

**Comment by verdurin** [comment]
@nathanhaigh Thanks for the contribution.

As it turns out, this PR stalled somewhat and in the meantime an equivalent was merged:

https://github.com/easybuilders/easybuild-easyconfigs/pull/9797

so on that basis I'm going to close this one.

This is mainly our fault for not keeping up, so please don't let this put you off...

**Pull request closed without merging**